### ISSUE TITLE
Fuel Station Cafe Hidden By Default 

### ISSUE THREAD

**Issue by pqhf5kd** [Rudloff/openvegemap#73]
https://www.openstreetmap.org/way/190602227

This may be something that should be discussed on the OSM Wiki/mailing list as well, but as in my example above, some fuel stations serve hot food. This maybe just a small takeaway counter or even restaurant style service. The OSM Wiki recommends adding a shop=yes tag but nothing regarding hot food. 

I assume OVM is hiding this example by default because it is a 'shop' but they actually serve hot vegan food (in this case a hotdog and a mushroom panini) which can be taken away or eaten at the tables inside. 

How do you suggest we deal with these cases?

**Comment by Rudloff** [comment]
Indeed, we hide it (unless you enable the *Shops* filter) because it is tagged as a shop.
IMHO, if the fuel station also contains a cafe, it should have an additional node with `amenity=cafe`.

**Comment by hermann-san** [comment]
IMO it would be nice to include  amenity=cafe as well, as a filter item.
Some of them are labeled vegan.  Like with vegan cake or other vegan bakery

**Comment by Rudloff** [comment]
We already display cafes: https://github.com/Rudloff/openvegemap/blob/fbd22ec61aa656179983c46c5311021d76b8b6ea/js/poi.js#L181

**Comment by hermann-san** [comment]
@Rudloff  strange, I can now also see cafe's. Sorry for bothering you. I don't know what happened.

Something else I've just noticed is that for [this Cafe](https://editor.openvegemap.netlib.re/node/3204496067 3204496067), the "Open in OSM" link doesn't work. It says "File not found"
